security-utils/src/main/java/com/yahoo/security/tls/HostnameVerification.java

```diff
@@ -0,0 +1,7 @@
+// Copyright 2020 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.security.tls;
+
+/**
+ * @author bjorncs
+ */
+public enum HostnameVerification { ENABLED, DISABLED }
```

Comment:
Should this be `HostnameValidation` to be in line with the config?

Reply:
The concept is called hostname verification in Java hence the name. I agree this could be renamed to be more in line with config name.

Reply:
It makes sense to use the concept that most closely maps to what's used on the underlying platform, so let's stick with verification for now.